prow/knative/plugins.yaml

```diff
@@ -94,6 +94,16 @@ config_updater:
 blunderbuss:
   use_status_availability: true
 
+# Settings for the "trigger" plugin.
+triggers:
+- repos:
+  - knative:
+    trigger_github_workflows: true
+  - knative-sandbox:
+    trigger_github_workflows: true
+  - knative-prow-robot:
```

Comment:
No need to add this since we used to add it for testing purpose. We can clean it up in follow-up PRs all together, though.